src/runtime/components/forms/SelectMenu.vue

```diff
@@ -413,8 +413,18 @@ export default defineComponent({
       }
 
       return (props.options as any[]).filter((option: any) => {
-        return (props.searchAttributes?.length ? props.searchAttributes : [props.optionAttribute]).some((searchAttribute: any) => {
-          return ['string', 'number'].includes(typeof option) ? option.toString().search(new RegExp(query.value, 'i')) !== -1 : (option[searchAttribute] && option[searchAttribute].search(new RegExp(query.value, 'i')) !== -1)
+        const searchable = typeof props.searchAttributes === 'string'
+          ? props.optionAttribute.split(',').map(attribute => attribute.trim())
```

Comment:
Why split the `optionAttribute` with `,`?

Reply:
You can do `:search-attributes="['value', 'text']"`, or `search-attributes="value,text"`